src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,8 @@
 #nullable enable
+NuGet.Commands.ResolvedDependencyKey.Equals(NuGet.Commands.ResolvedDependencyKey other) -> bool
+NuGet.Commands.ResolvedDependencyKey.ResolvedDependencyKey() -> void
+static NuGet.Commands.ResolvedDependencyKey.operator !=(NuGet.Commands.ResolvedDependencyKey left, NuGet.Commands.ResolvedDependencyKey right) -> bool
+static NuGet.Commands.ResolvedDependencyKey.operator ==(NuGet.Commands.ResolvedDependencyKey left, NuGet.Commands.ResolvedDependencyKey right) -> bool
+static NuGet.Commands.UnexpectedDependencyMessages.DependencyRangeHasMissingExactMatch(NuGet.Commands.ResolvedDependencyKey dependency) -> bool
```

Comment:
Should we just not make this method public?

If we're making a breaking change, might as well break it completely. This doesn't need to be public.

I'm guessing this was the issue that @jeffkl fixed, so I'm looking for @NuGet/nuget-client thoughts.

Reply:
`RestoreTargetGraph` exposes it at the moment: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreTargetGraph.cs#L57

Reply:
I was referring to the UnexpectedDependencyMessages method actually.


UnexpectedDependencyMessages does not need to be public at all.

Reply:
Ah sorry I thought you meant remove everything from public API.  I'll push a commit to remove this whole class from the public API and update the issue to indicate that the class was removed.